lib/gradeDialog/grade_dialog.dart

```diff
@@ -0,0 +1,190 @@
+                                    ? const Color(0xFF2DAD9D)
+                                    : const Color(0xFFFFF8C9),
+                                shape: const RoundedRectangleBorder(
+                                  borderRadius:
+                                      BorderRadius.all(Radius.circular(15.0)),
+                                ),
+                                child: Padding(
+                                  padding: const EdgeInsets.fromLTRB(
+                                      10.0, 20.0, 10.0, 10.0),
+                                  child: RichText(
+                                    textAlign: TextAlign.center,
+                                    text: TextSpan(children: [
+                                      TextSpan(
+                                        text: _list[index],
+                                        style: const TextStyle(
+                                          fontFamily: 'poppins',
+                                          fontSize: 28.0,
+                                          fontWeight: FontWeight.bold,
+                                          color: Color(0XFF5F5200),
+                                        ),
+                                      ),
+                                      WidgetSpan(
+                                        child: Transform.translate(
+                                          offset: const Offset(3, -12),
+                                          child: !lastIndex
+                                              ? const Text(
+                                                  'th',
+                                                  //superscript is usually smaller in size
+                                                  textScaleFactor: 1,
+                                                  style: TextStyle(
+                                                    fontSize: 14.0,
+                                                    color: Color(0XFF5F5200),
+                                                  ),
+                                                )
+                                              : null,
+                                        ),
+                                      )
+                                    ]),
+                                  ),
+                                ),
+                              ),
+                              Positioned(
+                                bottom: 12,
+                                right: 12,
+                                child: Offstage(
+                                  offstage: !checked,
+                                  child: Container(
+                                    decoration: const BoxDecoration(
+                                        color: Colors.white,
+                                        shape: BoxShape.circle),
+                                    child: const Icon(
+                                      Icons.check,
```

Comment:
there is icon called check_circle, use that.
Remove container and change the icon color from Color(0xFF2DAD9D) to white.
Set the icon size to 20

Reply:
added Icon(
                                    Icons.check_circle,
                                    color: Colors.white,
                                    size: 20.0,
                                  ),